### PR TITLE
Featur/ivashchenko#448 horizontal workshop

### PR DESCRIPTION
**Pull request by DenysIvashchenko**
create
createDeskTop
mainView
mobileView

### CODE REVIEW

File: src/app/shared/components/workshop-card/workshop-card.component.scss [hunk 1]
```diff
@@ -26,3 +26,65 @@
...
   }
 }
+
+.horizontal-card {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  overflow: hidden;
+  .card-img {
```
Comment (dmitryy90) [inline src/app/shared/components/workshop-card/workshop-card.component.scss:36]: indentations

File: src/app/shared/components/workshop-card/workshop-card.component.scss [hunk 2]
```diff
@@ -26,3 +26,65 @@
...
+    display: flex;
+    width: 100%;
+    min-width: 150px;
+    object-fit: cover;
+    object-position: bottom;
+  }
+  .mat-card-image{
+    width: 100%;
+    margin: 0;
+    margin-bottom: 0 !important;
```
Comment (dmitryy90) [inline src/app/shared/components/workshop-card/workshop-card.component.scss:46]: you can avoid using important by adding additional selector to increase priority

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

File: src/app/shared/components/workshop-card/workshop-card.component.html [hunk 3]
```diff
@@ -27,7 +29,7 @@
       <p class="card_text">
         <mat-icon class="card_icon">home</mat-icon>{{ workshop.providerTitle }}
       </p>
-      <p class="card_text">
+      <p class="card_text" [ngClass]="isCreateApplicationView ? 'hide': ''">
```
Comment (litvinets) [inline src/app/shared/components/workshop-card/workshop-card.component.html:32]: why should we hide this info?

Comment (DenysIvashchenko) [reply src/app/shared/components/workshop-card/workshop-card.component.html:32]: because 3 cards are different

**Review by litvinets — APPROVED** [review]

**Review by dmitryy90 — APPROVED** [review]

**Review by dmitryy90 — APPROVED** [review]

**Pull request merged**